### PR TITLE
fix(stripe): fix serialization issues on invoices

### PR DESCRIPTION
**Pull request by jeffladiray**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Create automatic tests
- [ ] No automatic tests failures
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by felipekb** [comment]
Hey, I think there's issues with a single invoice serialization too, I'm getting  `NoMethodError (undefined method 'date' for #<Stripe::Invoice ` when trying to display a stripe invoice

**Comment by jeffladiray** [comment]
Hi :wave: 

Really sorry about the delayed response.
I just pushed a fix for that as well, and I manually tested all collections/individual model, just to make sure there were no similar issue on this. If possible on your end, a quick test for this fix would be helpful :pray: 

Thanks in advance

File: app/serializers/forest_liana/stripe_invoice_serializer.rb [hunk 1]
```diff
@@ -3,20 +3,20 @@ class StripeInvoiceSerializer
     include JSONAPI::Serializer
 
     attribute :amount_due
+    attribute :amount_paid
```
Comment (VincentMolinie) [inline app/serializers/forest_liana/stripe_invoice_serializer.rb:6]: Doesn't those depends on the stripe version ? What if someone use on old version ofStripe ?

**Comment by felipekb** [comment]
> I just pushed a fix for that as well, and I manually tested all collections/individual model, just to make sure there were no similar issue on this. If possible on your end, a quick test for this fix would be helpful 🙏

@jeffladiray Tested for an invoice and it's working, Thanks!

**Review by guillaumedeslandes — APPROVED** [review]
🚀

**Pull request merged**